Source/core/IPCConnector.h

```diff
@@ -331,59 +331,55 @@ namespace Core {
             // -----------------------------------------------------
             HAS_MEMBER(Clear, hasClear);
 
-            typedef hasClear<PACKAGE, void (PACKAGE::*)()> TraitClear;
 
-            template <typename SUBJECT, const uint32_t ID>
-            inline typename Core::TypeTraits::enable_if<RawSerializedType<SUBJECT, ID>::TraitClear::value, void>::type
+            template <typename SUBJECT=PACKAGE>
+            inline typename Core::TypeTraits::enable_if<hasClear<SUBJECT, void (SUBJECT::*)()>::value, void>::type
             __Clear()
             {
                 _package.Clear();
             }
 
-            template <typename SUBJECT, const uint32_t ID>
-            inline typename Core::TypeTraits::enable_if<!RawSerializedType<SUBJECT, ID>::TraitClear::value, void>::type
+            template <typename SUBJECT=PACKAGE>
+            inline typename Core::TypeTraits::enable_if<!hasClear<SUBJECT, void (SUBJECT::*)()>::value, void>::type
             __Clear()
             {
             }
 
             // -----------------------------------------------------
             // Search for custom handling, Compile time !!!
             // -----------------------------------------------------
-            HAS_MEMBER(Length, hasLength);
+            HAS_MEMBER_NAME(Length, hasLength);
```

Comment:
Hi Wissem, why did you now change this to HAS_MEMBER_NAME? Didn't it compile with HAS_MEMBER?

Reply:
good catch. Sorry, I was testing with it. and committed it by mistake.reverted.